components/card/card-footer-link.js

```diff
@@ -10,7 +11,7 @@ import { RtlMixin } from '../../mixins/rtl-mixin.js';
  * An icon link that can be placed in the `footer` slot.
  * @slot tooltip - slot for the link tooltip
  */
-class CardFooterLink extends RtlMixin(LitElement) {
+class CardFooterLink extends FocusMixin(RtlMixin(LitElement)) {
```

Comment:
This and collapsible panel were in a situation where previously they were relying on the fact that `forceFocusVisible` finds the first focusable descendent to focus on, whereas now we simply call `focus()` on the element itself. By implementing `FocusMixin`, the components can delegate focus, which solves the problem.